### ISSUE TITLE
Listing / Reporting

### ISSUE THREAD

**Issue by tobinharris** [alisaifee/jira-cli#34]
Hi there

Nice tool. Is there a simple way to list all open issues with status?

**Comment by alisaifee** [comment]
Yes, you should be able to use the jql search option. (`jira-cli view --search-jql='...'`)
